src/EventsFeedPage/css/event-info.css

```diff
@@ -28,14 +31,17 @@
 
 #event-desc {
   margin-top: 10px;
+  color: #9b1522;
 }
 
 #event-pic {
   position: fixed;
+  justify-content: center;
 }
 
 #event-pic img {
-  width: 100%;
+  width: 50%;
+
```

Comment:
I'd remove this extra newline. It's no big deal, just makes the code a tiny bit harder to parse when you're scanning.